web-marketplace/src/pages/BuyCredits/BuyCredits.Form.tsx

```diff
@@ -130,7 +145,7 @@ export const BuyCreditsForm = ({
     }),
   );
 
-  const stripePromise = loadStripe(stripeKey);
+  const stripe = useMemo(() => stripePromise, []);
```

Comment:
Maybe we can use `stripePromise` directly as it's declared outside the component and will not change between renders.